### PR TITLE
feat: implement ErrorPage component

### PR DESCRIPTION
**Pull request by sofyahreksoftware**
I have implemented an **ErrorPage component**. To see how it looks, you need to add the following code to `App.js`

```js
import { ErrorPage } from "./ErrorPage";

const App = () => <ErrorPage />;

export default App;

### CODE REVIEW

**Comment by wroblewski82** [comment]
Hi Sofia.
I believe that giving up the use of Flex and applying block-by-block styling will greatly simplify the code.
In the screenshots I presented my styling proposal.
The StyledLink component uses the "Open Sans" font.

File: src/ErrorPage/styled.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+export const Container = styled.div`
+  padding: 16px;
+  text-align: center;
+  margin: 195px auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 24px;
```
Comment (wroblewskisoftware) [inline src/ErrorPage/styled.js]: I think we can remove all this code from the Container component.

File: src/ErrorPage/styled.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  padding: 16px;
```
Comment (wroblewskisoftware) [inline src/ErrorPage/styled.js]: Then change the padding in the Container component to padding: 180px 16px 32px;

File: src/ErrorPage/styled.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    margin: auto;
+  }
+`;
+
+export const StyledHeader = styled.h1`
+  font-weight: 600;
+  font-size: 36px;
+  line-height: 43.2px;
+  padding: 0 0 120px;
+  margin: 0;
```
Comment (wroblewskisoftware) [inline src/ErrorPage/styled.js]: Then remove this part of the code from the StyledHeader component.

File: src/ErrorPage/styled.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+`;
+
+export const StyledParagraph = styled.p`
+  max-width: 426.53px;
+  font-weight: 500;
+  font-size: 22px;
+  line-height: 28.6px;
+  padding: 2px 0;
+  text-align: center;
```
Comment (wroblewskisoftware) [inline src/ErrorPage/styled.js]: Then remove this part of the code from the StyledParagraph component.

File: src/ErrorPage/styled.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  margin: -76px auto 24px;
```
Comment (wroblewskisoftware) [inline src/ErrorPage/styled.js]: Then change the margin in the StyledParagraph component to margin: 0 auto 40px;

File: src/ErrorPage/styled.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+`;
+
+export const StyledLink = styled.a`
```
Comment (wroblewskisoftware) [inline src/ErrorPage/styled.js:26]: In the StyledLink component we must use the "Open Sans" font.

**Review by wroblewskisoftware — CHANGES_REQUESTED** [review]
Hi Sofia.
I believe that giving up the use of Flex and applying block-by-block styling will greatly simplify the code.
I sent the effect of the above-mentioned changes in a private message on Slack.
The StyledLink component uses the "Open Sans" font.

File: src/ErrorPage/styled.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  padding: 180px 16px 32px;
+  text-align: center;
+
+  @media (max-width: ${({ theme }) => theme.media.mobile}px) {
+    margin: auto;
+  }
```
Comment (wroblewskisoftware) [inline src/ErrorPage/styled.js:9]: I think that in this case the margin would be better replaced by padding: 16px 16px 32px;
This gives a nicer effect on mobile devices.

File: public/index.html [hunk 8]
```diff
@@ -22,16 +21,24 @@
...
+    <title>Movies Browser</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
```
Comment (wroblewskisoftware) [inline public/index.html:33]: Only one font weight of 700 and no Italic can be imported here.

**Review by wroblewskisoftware — APPROVED** [review]
Good job. You can merge.

File: src/ErrorPage/styled.js (hunk 7 above)
Comment (sofyahreksoftware) [reply src/ErrorPage/styled.js:9]: In my opinion, it looks better on mobile, when it's vertically centered:)
Layout at the top:
layout at the top
and vertically centred:
layout vertically centred

**Pull request merged**